docs/dev/news/3660.feat

```diff
@@ -0,0 +1 @@
+The `dafny audit` command now reports instances of the `{:concurrent}` attribute, intended to flag code that is intended, but can't be proven, to be safe for use in a concurrent setting.
```

Comment:
```suggestion
The `dafny audit` command now reports instances of the `{:concurrent}` attribute, intended to flag code that is intended to be safe for use in a concurrent setting.
```

"but can't be proven" implies there is could that could be proven safe, but Dafny can't do that yet.